bin/createteams.py

```diff
@@ -1,14 +1,17 @@
-# NOTE: click is not a dependency of this project, you'll have to manually
-#       install it to run this file.
-import click
+import sys
+try:
+    import click
+except ImportError:
+    sys.exit("""You have to manually click to run this file.""")
```

Comment:
"click" -> "install the 'click' package"

Reply:
Why the triple quotes?

Reply:
No good reason. I originally had it as a multiple line string. Both things  have been addressed